eslint-plugin-prettier.js

```diff
@@ -313,15 +331,26 @@ module.exports = {
               { type: 'object', properties: {}, additionalProperties: true }
             ]
           },
-          // Pragma:
-          { type: 'string', pattern: '^@\\w+$' }
+          {
+            anyOf: [
+              // Pragma:
+              { type: 'string', pattern: '^@\\w+$' },
+              {
+                type: 'object',
+                properties: {
+                  pragma: { type: 'string', pattern: '^@\\w+$' },
+                  usePrettierRc: { type: 'boolean' }
```

Comment:
Can you add a test for the new option? (It would probably be similar to the existing `prettierrc` tests, except that the test would be asserting that the options in the file do not get applied.)